packages/create-mc-app/src/logger.js

```diff
@@ -1,9 +1,13 @@
-const signale = require('signale');
+const { Signale } = require('signale');
 
-signale.config({
-  displayBadge: false,
+module.exports = new Signale({
+  config: { displayBadge: false, displayLabel: false },
```

Comment:
I find it a bit more clean without these, as we already have the "scope" next to the log

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/1110551/50820054-067f3700-132c-11e9-86cf-1568102f3f72.png">